modules/calc/src/main/java/com/opengamma/strata/calc/runner/function/CalculationFunction.java

```diff
@@ -83,6 +86,20 @@
   public abstract Currency naturalCurrency(T target, ReferenceData refData);
 
   /**
+   * Manages the calculation tasks.
+   * <p>
+   * This allows a function to control the tasks that are executed to calculate the results.
+   * <p>
+   * The tasks are created using a dedicated builder that provides access to the target and columns.
+   * Customization of the tasks can be achieved by calling methods on the builder.
+   * 
+   * @param builder  the calculation task builder
+   */
+  public default void manageTasks(TaskBuilder builder) {
```

Comment:
I'm not keen on this for a number of reasons.

* `manage` is a non-name that doesn't communicate anything about what's supposed to be happening
* The signature means there must be mutation going on which is something we've tried to avoid in Strata

Presumably this only exists to allow the function to call `builder.executeSeparately`? Isn't there a better way to achieve the same thing?